src/HealthChecks.Publisher.ApplicationInsights/ApplicationInsightsPublisher.cs

```diff
@@ -16,17 +16,20 @@ internal class ApplicationInsightsPublisher : IHealthCheckPublisher
         private static TelemetryClient? _client;
         private static readonly object _syncRoot = new object();
         private readonly TelemetryConfiguration? _telemetryConfiguration;
+        private readonly string? _connectionString;
         private readonly string? _instrumentationKey;
         private readonly bool _saveDetailedReport;
         private readonly bool _excludeHealthyReports;
 
         public ApplicationInsightsPublisher(
             IOptions<TelemetryConfiguration>? telemetryConfiguration,
+            string? connectionString = default,

```

Comment:
Technically this is a binary breaking change.

Reply:
Should I list `connectionString` after `instrumentationKey`? Or what do you suggest?

Reply:
Oops, sorry, I missed `internal` access modifier. Not a subject of public API surface.